app/workers/rapid_ready_for_decision/form526_base_job.rb

```diff
@@ -72,6 +72,11 @@ def perform(form526_submission_id)
       end
     end
 
+    # Override this method to prevent the submission from getting the PDF and special issue
+    def release_pdf?(_form526_submission)
```

Comment:
I'm tempted to turn `form526_submission` into an instance variable to avoid having to pass it around to these methods. Thoughts? My current thinking: this class is instantiated by `Sidekiq::Batch`; `perform(form526_submission_id)` is called -- we can set `@form526_submission` at the beginning of `perform` and unset at the end. The potential problem is if Sidekiq reuses the job instance on another `form526_submission_id` concurrently -- is this possible?
Another approach is to create another class that is instantiated with a `form526_submission` and move most of the methods there. I think I prefer this approach.

Reply:
These are good topics for our weekly engineers' discussion!

Initial thoughts: agreed, even if I doubt Sidekiq would ever concurrently reuse job instances, it's better to not tie our implementation to that assumption. So maybe the `process_selector` can pass both a Form526Submission ID and a classname over to a whittled-down `#perform` that does the actual instantiation of the subclass?